doc/contrib/README.md

```diff
@@ -12,6 +12,18 @@ If you find a bug in the code or a mistake in the documentation, you can help us
 
 ## Development
 
+### Hooks
```

Comment:
We should either mention hooks in .github/CONTRIBUTING.md or even better: link [doc/contrib/README.md](https://github.com/ElektraInitiative/PermaplanT/pull/160/files#diff-d5f5656eaeb7cfba734b6f3046c390588e849c59f79f74d1de13067af6a92f58) from there.

Reply:
see
https://github.com/ElektraInitiative/PermaplanT/pull/160/commits/38e26b095b5ff3f665af19ec9d4a5a2078e254bc
https://github.com/ElektraInitiative/PermaplanT/pull/160/commits/6c983bb656c708b0f1f51a5ab049c6819d55b9a9